app/controllers/finance/api/invoices_controller.rb

```diff
@@ -9,10 +9,8 @@ class Finance::Api::InvoicesController < Finance::Api::BaseController
   # Invoice List
   # GET  /api/invoices.xml
   def index
-    search = ThreeScale::Search.new(params[:search] || params)
-    results = invoices.scope_search(search)
-                .order_by(params[:sort], params[:direction])
-                .paginate(pagination_params)
+    search = ThreeScale::Search.new(params)
+    results = invoices.scope_search(search).paginate(pagination_params)
```

Comment:
so, `sort`, `direction` and `search` params are not documented, and probably don't make sense.

Even when I try to pass `sort` and `direction`, it doesn't work because of `ordered_relation(relation)` in the responder, that reorders by `:id`.

Reply:
Maybe it was supposed to work like in app/presenters/finance/provider/invoices_index_presenter.rb :thinking: 

But since is didn't, I guess we arefine to leave it as is for the time being.